solr/solrj/src/java/org/apache/solr/common/cloud/ClusterState.java

```diff
@@ -400,29 +402,14 @@ public Set<String> getHostAllowList() {
     return hostAllowList;
   }
 
-  /**
-   * Iterate over collections. Unlike {@link #getCollectionStates()} collections passed to the
-   * consumer are guaranteed to exist.
-   *
-   * @param consumer collection consumer.
-   */
+  /** Streams the resolved DocCollections. Use this sparingly in case there are many collections. */
+  public Stream<DocCollection> collectionStream() {
+    return collectionStates.values().stream().map(CollectionRef::get).filter(Objects::nonNull);
+  }
+
+  /** Streams the resolved DocCollections. Use this sparingly in case there are many collections. */
```

Comment:
Seems that Javadoc comment was copied from above method and needs updating

Reply:
It was deliberate as I thought it was good enough, but I'll add more words